### ISSUE TITLE
Support property assignment on constant objects 

### ISSUE THREAD

**Issue by Bodacious** [Bodacious/no-magic-numbers-cop#1]
## What 

Do not report a violation when assigning magic numbers on class methods. 

``` ruby 
# No offence
Foo.bar = 1
```

## Why

This pattern is typically used to configure a library. The setter methods are usually descriptive enough that we don't need to consider them "magic numbers"
